docs/comparison-matrix.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-file -->
+
+# Comparison Matrix 
+
+Legend:
+
+- 🟢 Available (grading added in some cases spanning: Bad, Average, Good, Excellent)
+- 🟡 In development
+- 🔵 Potential future feature
+- 🔴 Not currently available
+- ❔ Not clear
+
+| **Feature**                                            | **Proto Buffers** | **ADL**      | **JSON Schema** | **Lambda Buffers** | **CDDL** | **ASN.1**    |
```

Comment:
Nothing about typeclasses / function generation? I think we should mention that here, it's supposed to be one of our killer features is it not?

Reply:
It is:

| Add new type semantics (e.g. different encodings)|
| Manage type semantics (at language level)

Reply:
We need to reiterate on this and add sub chapters elaborating on what each listed feature actually means.

Reply:
I don't think they are clear, was hoping you'll bring out better vocab

Reply:
Resolving this, Vlad is expanding on the meaning of these features, hopefully that will help.